app/models/order.py

```diff
@@ -121,6 +122,7 @@ def __init__(self,
         self.deleted_at = deleted_at
         self.order_notes = order_notes
         self.tickets_pdf_url = tickets_pdf_url
+        self.paypal_token = paypal_token
```

Comment:
Why is this needed?

Reply:
so that I can create an Order to save paypal payment ID. @mrsaicharan1 commented above that I can find paypal payment ID in Order model to save it, but I couldn't find it Order model so I thought he meant paypal_token.

Reply:
@mrsaicharan1 Please comment

Reply:
@anhanh11001 This is not required. So just verify the payment in the way I've commented

Reply:
Orders should never be created with a paypal_token. Consider this as imitating a paid order through paypal. After an order is created, only after proper authorisation from PayPal's API, the attribute should be updated. 
All you need to do is verify it right? So please remove this.

Reply:
This is not needed anywhere AFAIK